geotrek/trekking/tests/test_views.py

```diff
@@ -120,6 +121,14 @@ def test_status_only_review(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'Waiting for publication')
 
+    @override_settings(ALERT_REVIEW=True)
+    def test_status_review_alert(self):
+        element_not_published = self.modelfactory.create()
+        element_not_published.published = False
```

Comment:
Je l'aurais plutôt mis dans la factory et enlevé cette ligne
```suggestion
        element_not_published = self.modelfactory.create(published=False)
```

Reply:
+ ajouter le changelog